common/network-common/src/main/java/org/apache/spark/network/server/OneForOneStreamManager.java

```diff
@@ -57,7 +57,7 @@
     int curChunk = 0;
 
     // Used to keep track of the number of chunks being transferred and not finished yet.
-    volatile long chunksBeingTransferred = 0L;
+    volatile AtomicLong chunksBeingTransferred = new AtomicLong(0L);
```

Comment:
No longer needed to be `volatile`, as well as can add `final` to make sure we don't change assignment.

Reply:
Done